themes/Rozier/AjaxControllers/AjaxDocumentsExplorerController.php

```diff
@@ -119,7 +116,92 @@ public function indexAction(Request $request)
             Response::HTTP_OK
         );
     }
+
+    /**
+     * Get a Document list from an array of id.
+     *
+     * @param Request $request
+     * @return JsonResponse
+     */
+    public function listAction(Request $request)
+    {
+        /*
+         * Validate
+         */
+        if (true !== $notValid = $this->validateRequest($request, 'GET')) {
```

Comment:
Remove `$this->validateRequest($request, 'GET')` to centralize verb validation in YAML routes with 

```yaml
methods: [GET]
```